.github/workflows/bump-version-release.yml

```
@@ -234,6 +234,43 @@ jobs:
         if: steps.changed-files-map-template.outputs.any_changed == 'true'
         run: npm -w @mapsindoors/map-template publish --access public
 
+  components-docs-release:
+    name: Release Components Docs
+    needs: components-release-npm
+    runs-on: ubuntu-latest
+    if: ${{needs.components-release-npm.outputs.has_files_changed}} == 'true'
```

actionlint (1.7.12)

[error] if: condition "${{needs.components-release-npm.outputs.has_files_changed}} == 'true'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)